### ISSUE TITLE
dll location may prefer broken system libraries

### ISSUE THREAD

**Issue by plicease** [uperl/Archive-Libarchive#40]
we automatically install Archive::Libarchive3 if the system libarchive is not sufficiently good enough, but the way the FFI::CheckLib logic works it might still pick the older system library.

Probably needs to be fixed in FFI::CheckLib.  Though could be fixed in here.

**Comment by wbraswell** [comment]
I think this is the pertinent paste:
http://paste.scsys.co.uk/594
Per the direction of @plicease I was able to get it to work with the following commands:
```
perl -MAlien::Libarchive3 -E 'say for Alien::Libarchive3->dynamic_libs'
export ARCHIVE_LIBARCHIVE_LIB_DLL=/full/path/to/libarchive.so
```

**Comment by plicease** [comment]
This should be fixed in #43
